lib/core/macros.nim

```diff
@@ -1626,9 +1626,18 @@ macro unpackVarargs*(callee: untyped; args: varargs[untyped]): untyped =
     result.add args[i]
 
 proc getProjectPath*(): string = discard
-  ## Returns the path to the currently compiling project, not to
-  ## be confused with ``system.currentSourcePath`` which returns
-  ## the path of the current module.
+  ## Returns the path to the currently compiling project,
+  ## the path of the Nim file being compiled,
+  ## not to be confused with ``system.currentSourcePath`` which returns
+  ## the path of the current module. Its available at compile-time.
+  ## Its available for JavaScript, NodeJS and NimScript backends.
+  ## Its available for C, C++ and ObjectiveC backends.
+  ##
+  ## See also:
+  ## * `getHomeDir proc <https://nim-lang.org/docs/os.html#getHomeDir>`_
```

Comment:
If linking within Nim's documentation, it's better to use relative links for two main reasons:
* It's friendlier to those browsing the docs offline. Many Linux distros have documentation packages such as Debian's `nim-doc` package that enables this use case, and some users may have built the docs themselves.
* If someone is browsing the devel docs, clicking that link will send them to the stable version of the docs instead, which may be different enough to cause confusion.

Thus, the proper link would look like: `os.html#getHomeDir`
This applies to similar links as well.